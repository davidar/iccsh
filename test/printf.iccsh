:declare x int x = 4
:declare y int y = 2
:printf x=%d,y=%d x, y
%d x+y
