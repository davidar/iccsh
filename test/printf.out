x=4,y=2
6
